src/fun/builtins.bend

```diff
@@ -175,3 +175,120 @@ hvm to_u24:
 # Casts any native number to an i24.
 hvm to_i24:
   ($([i24] ret) ret)
+
+# Encoding
+
+Utf8/MAX_RUNE = '\u{0010FFFF}'
+Utf8/RUNE_ERROR = '\u{FFFD}'
+#Utf8/RUNE_SELF = 0x80
+#Utf8/MAX = 4
+
+Utf8/t1 = 0b00000000
+Utf8/tx = 0b10000000
+Utf8/t2 = 0b11000000
+Utf8/t3 = 0b11100000
+Utf8/t4 = 0b11110000
```

Comment:
Can you convert these 4 into `use` terms? We should avoid polluting the builtins with functions that are not useful for users.